jumanji/environments/logic/minesweeper/env.py

```diff
@@ -181,8 +164,8 @@ def step(
             next_timestep: `TimeStep` corresponding to the timestep returned by the environment.
         """
         board = state.board
-        action_height, action_width = action
-        board = board.at[action_height, action_width].set(
+        action_row, action_col = action
+        board = board.at[action_row, action_col].set(
             count_adjacent_mines(state=state, action=action)
         )
```

Comment:
Suggestion:
```suggestion
        board = state.board.at[tuple(action)].set(
            count_adjacent_mines(state=state, action=action)
        )
```
In this case you can even remove the above line `board = state.board`.